server/tournament.py

```diff
@@ -3,21 +3,23 @@
 from django.db.models import Q
 
 from server.models import Bracket, CrossPool, Match, Pool, PositionPool, Team, Tournament
+from server.utils import get_bracket_match_name
 
 # Exported Functions ####################
 
 
 def create_pool_matches(tournament: Tournament, pool: Pool) -> None:
     pool_seeding_list = list(map(int, pool.initial_seeding.keys()))
 
-    for i, x in enumerate(pool_seeding_list):
-        for _j, y in enumerate(pool_seeding_list[i + 1 :], i + 1):
+    for i, seed_x in enumerate(pool_seeding_list):
```

Comment:
We should do the same in Position Pool I think

Reply:
Yeah, same names logic as position pool. I'll make these updates

Reply:
also, are position pools the same as pools ? code looks to be the same

Reply:
yep its exactly the same

Reply:
> We should do the same in Position Pool I think
name like "Position pool <pool name> seed_1 v seed _2" sounds good ?

Reply:
Just K1 v K2 should be good here... it wont be K9 K11 lol.. number will be relative to the pool

Reply:
Yeah lol mb it will be relative. On schedule UI, how will we differentiate position pool from pool ? Different colours or prefix of "position pool" for the name ?